test/integration/test_driver.go.tmpl

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
This file should have the imports and stand on its own (beside `GetOpenAPIDefinitions` which is coming from the generate file)

Reply:
Right. I don't understand why this is a .tmpl file instead of a .go file.